app/models/tslr_claim.rb

```diff
@@ -46,7 +57,9 @@ class TslrClaim < ApplicationRecord
 
   validates :employment_status,         on: [:"still-teaching", :submit], presence: {message: "Choose the option that describes your current employment status"}
 
-  validates :mostly_teaching_eligible_subjects, on: [:"subjects-taught", :submit], inclusion: {in: [true, false], message: "Select either Yes or No"}
+  validates :eligible_subjects,         on: [:"subjects-taught", :submit], presence: {message: "Choose a subject, or select \"not applicable\""}, unless: ->(c) { c.mostly_teaching_eligible_subjects == false }
```

Comment:
If changing the "no" answer to the question, the error message will also need updating.